Pod/Classes/UIControlBlocks/UIControl+Blocks.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  UIControl+Blocks.swift
+//  Base
+//
+//  Created by Brian King on 6/23/17.
+//
+//
+
+import UIKit
+
+public protocol BlockControl {}
+
+extension UIControl: BlockControl {}
+
+public extension BlockControl where Self: UIControl {
+    /**
+     Set a block handler for an event.
+
+     - parameter event: A UIControlEvents event
+     - parameter callback: The event handler function
+     */
+    func setBlockTarget(for event: UIControlEvents, callback: @escaping (Self) -> Void) {
```

Comment:
#pp this notation seems outdated. RE: Closures vs. blocks. how about `setCallback(for _, callback: _)`

Reply:
`setCallback(for event: UIControlEvent, _ callback: ...)` maybe? I don't like the double `callback` in the signature.

Reply:
that looks good